content/en/docs/refguide/modeling/pages/page/_index.md

```diff
@@ -94,10 +94,38 @@ In **Structure mode**, there are several ways to add an element on a page:
 
     1. Select an element you would like to add and confirm your choice by clicking **Select**.
 
-In **Design mode**, you can add elements though the Toolbox. Do the following:
+In **Design mode**, you can add elements through the Toolbox and UI Recommender. Do the following:
 
-1. Open the **Toolbox** and select **Widgets** or **Building blocks** tab there. 
-2. Select an element you would like to add and drag this element onto your page.
+1. Through the **Toolbox**:
+   1. Open the **Toolbox** and select **Widgets** or **Building blocks** tab there. 
+   1. Select an element you would like to add and drag this element onto your page.
+
+2. Through the **UI Recommender**:
+    1. Opening the Dialog
+
+        1. When hovering over an element on the page, a blue plus/dot will appear:
+            1. The blue dot will appear at the closest border of the widget (top/bottom or left/right for inline widgets).
+            1. **Widgets** can be inserted before or after an element: top/left for before, bottom/right for after.
```

Comment:
I think here you can say that "The position of the indicator relates to where an element will be inserted, i.e. selecting the indicator at the top or left will insert an element before the current widget"

Reply:
Updated